test/e2e_node/runner/remote/run_remote.go

```diff
@@ -497,24 +488,23 @@ func getGCEImages(imageRegex, imageFamily string, project string, previousImages
 					creationTime: creationTime,
 					name:         instance.Name,
 				}
-				klog.V(4).Infof("Found image %q based on regex %q and family %q in project %q", io.string(), imageRegex, imageFamily, project)
 				imageObjs = append(imageObjs, io)
 			}
 			return nil
 		},
 	); err != nil {
 		return nil, fmt.Errorf("failed to list images in project %q: %v", project, err)
 	}
+
+	// Pick the latest image after sorting.
 	sort.Sort(byCreationTime(imageObjs))
 	images := []string{}
-	for _, imageObj := range imageObjs {
-		images = append(images, imageObj.name)
-		previousImages--
-		if previousImages < 0 {
-			break
-		}
+	if len(imageObjs) > 0 {
+		klog.V(4).Infof("found images %+v based on regex %q and family %q in project %q", imageObjs, imageRegex, imageFamily, project)
+		images = []string{imageObjs[0].name}
+		return images, nil
```

Comment:
Since this will always return the latest image, might as well just change this to
```go
return imageObjs[0].name, nil
```